Loop/Managers/Alerts/AlertStore.xcdatamodeld/AlertStore.xcdatamodel/contents

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="17511" systemVersion="20C69" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="19574" systemVersion="21A559" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
     <entity name="StoredAlert" representedClassName=".StoredAlert" syncable="YES">
         <attribute name="acknowledgedDate" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
         <attribute name="alertIdentifier" attributeType="String"/>
         <attribute name="backgroundContent" optional="YES" attributeType="String"/>
         <attribute name="foregroundContent" optional="YES" attributeType="String"/>
-        <attribute name="isCritical" attributeType="Boolean" usesScalarValueType="YES"/>
+        <attribute name="interruptionLevel" attributeType="Integer 16" usesScalarValueType="YES"/>
```

Comment:
Update the the mappingmodel?

Reply:
Ah, thanks for the reminder!

Reply:
Ah wait, there is no mappingmodel, because this store doesn't exist out there in released form, so there's no need (yet) for a migration.

Reply:
So it doesn't even need a placeholder in the mappingmodel file?

Reply:
There is no mappingModel file.

Reply:
Should we version this and create one? I believe all of this is in DIY now, at least in the `dev` branch.